build.gradle

```diff
@@ -85,6 +85,9 @@ ext {
   userMaxTestRetries = project.hasProperty('maxTestRetries') ? maxTestRetries.toInteger() : 0
   userMaxTestRetryFailures = project.hasProperty('maxTestRetryFailures') ? maxTestRetryFailures.toInteger() : 0
 
+  userMaxQuarantineTestRetries = project.hasProperty('maxQuarantineTestRetries') ? maxQuarantineTestRetries.toInteger() : 0
```

Comment:
Could you please add it to `README`?

https://github.com/apache/kafka/blob/trunk/README.md#specifying-test-retries